app/helpers/admin_helper.rb

```diff
@@ -54,7 +54,16 @@ def admin_custom_colors
   end
 
   def format_currency(amount, currency, tooltip: true)
-    display_symbol = currency ? currency.symbol.sub('$', ' $') : '' # add space before $ (pretty)
+    # add space before $ and C$ (pretty)
```

Comment:
I think I would probably remove the changes to this function and just add the space in the DB, since that reduces the amount of special-case formatting we have to do in this function.

Reply:
Otherwise this looks good!